web2/htdocs/index.html

```diff
@@ -167,7 +167,32 @@ <h2>You Are About To Restore Data</h2>
               <button class="closes danger" rel="yes">Yes, Initiate Restore</button>
             </div>
           </div>
+          
     <!-- }}} --></script>
+
+    <script type="text/html" id="template:delete-are-you-sure"><!-- {{{ -->
+      [[#
+        {delete-are-you-sure}
+
+        A modal interaction screen that requires the operator to acknowledge
+        that what they are about to do is in fact to delete data, that it
+        might be dangerous, and it might not work out.
+      ]]
+          <div class="confirm">
+            <h2>You Are About To Delete Your Backup Job</h2>
+
+            <p>You have requested that SHIELD delete <em>[[= _.name]]</em>.
+            Backups can no longer be created from this job when destroy.</p>
```

Comment:
"from this job when destroy"?

Reply:
What happens to archives that already exist that were made by this job?  Probably should call that out.